### ISSUE TITLE
doctrine/migrations bad namespace

### ISSUE THREAD

**Issue by mychalvlcek** [symfony/recipes#100]
when you use default configuration of migrations bundle, it uses `Application\Migrations` namespace for migration files.. but new src structure of symfony > 3.3 requires namespace `App\Migrations`

so i recommend use following cfg of this recipe

```
// doctrine/doctrine-migrations-bundle/1.2/etc/packages/doctrine_migrations.yaml
doctrine_migrations:
    dir_name: '%kernel.project_dir%/src/Migrations'
    namespace: App\Migrations

```

**Comment by xabbuh** [comment]
there is an ongoing discussion in #87

**Comment by fabpot** [comment]
fixed by #172
